roxie/ccd/ccdmain.cpp

```diff
@@ -374,6 +374,7 @@ class MAbortHandler : implements IExceptionHandler
         {
             socketListeners.item(idx).stopListening();
         }
+        kill(getpid(), SIGABRT);
```

Comment:
I do not see the need for this.  Without it we still get a core file from a SEGV.